webpack.config.js

```diff
@@ -54,7 +54,7 @@ module.exports = ( env ) => {
      * See: https://webpack.js.org/configuration/devtool/#devtool
      * See: https://github.com/webpack/docs/wiki/build-performance#sourcemaps
      */
-    devtool: ifProd( 'source-map', 'cheap-module-source-map' ),
+    devtool: 'source-map',
```

Comment:
yes, but actually this shouldn't be part of this PR, I will create new PR with issue